sql-db/vertx-sql/src/test/java/io/quarkus/qe/vertx/sql/dbpool/PostgresPoolIT.java

```diff
@@ -0,0 +1,153 @@
+package io.quarkus.qe.vertx.sql.dbpool;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.greaterThanOrEqualTo;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.MethodOrderer.OrderAnnotation;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+import io.quarkus.test.bootstrap.DefaultService;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.Container;
+import io.quarkus.test.services.QuarkusApplication;
+import io.restassured.response.Response;
+import io.smallrye.mutiny.Uni;
+import io.vertx.core.json.JsonArray;
+import io.vertx.ext.web.client.WebClientOptions;
+import io.vertx.mutiny.core.Vertx;
+import io.vertx.mutiny.ext.web.client.HttpResponse;
+import io.vertx.mutiny.ext.web.client.WebClient;
+import io.vertx.mutiny.ext.web.client.predicate.ResponsePredicate;
+
+@QuarkusScenario
+@TestMethodOrder(OrderAnnotation.class)
+public class PostgresPoolIT {
+    private static final String POSTGRESQL_DATABASE = "amadeus";
+
+    static final int TIMEOUT_SEC = 60;
+    static final int HTTP_OK = 200;
+    static WebClient httpClient;
+
+    @ConfigProperty(name = "quarkus.datasource.reactive.max-size")
+    int datasourceMaxSize;
```

Comment:
This is not working... (not supported). It always returns 0.

Reply:
Changed to constant